src/main.py

```diff
@@ -31,17 +31,16 @@ def main(configfile):
     log.addHandler(ch)
 
     try:
+        log.info('Starting up')
         while True:
-            log.info('Starting up')
             processor.go(c)
             time.sleep(60)
     except KeyboardInterrupt:
         log.info('Exiting on user interrupt')
         sys.exit(0)
-    except:
-        log.error('Exception')
-        raise util.CoastguardException
-
+    except CoastguardAPIError:
```

Comment:
For a long-running process like this you'll want to handle API exceptions intelligently, since they will happen now and then. Logging is usually sufficient since the next check is happening in a minute anyway.  But for now, this is fine.

Of course, if you have something auto-restarting the checker when it dies, then dying here isn't so bad either.